documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/http/server-http-request-logging.adoc

```diff
@@ -0,0 +1,48 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+[[pg-server-http-request-logging]]
+==== Request Logging
```

Comment:
Should we talk about the `CustomRequestLog.LogDetail` record here?   It is needed to communicate details from the servlet environments.    I don't like the mechanism very much... mostly because it is not extensible and very arbitrary.

Perhaps we should improve it first and then document it?

Reply:
I just moved this section as is. If it needs additions, should be done in a separate PR. Want to open an issue about this?

Reply:
Will do.